src/ts_catalog/catalog.h

```diff
@@ -1484,6 +1486,10 @@ typedef struct FormData_job_error
 
 typedef FormData_job_error *Form_job_error;
 
+#define HYPERTABLE_STATUS_DEFAULT 0
+#define HYPERTABLE_STATUS_OSM 1
+#define HYPERTABLE_STATUS_OSM_CHUNKS_OVERLAP 2
```

Comment:
This is now no longer true since overlap is an illegal state. Should we rename the flag to better reflect the situation?
```suggestion
#define HYPERTABLE_STATUS_OSM_CHUNK_NONCONTIGUOUS 2
```

Reply:
updated